server/service/individualServices/AuthorizationApplication.js

```diff
@@ -254,7 +252,7 @@ exports.IsApplicationExists = async function (applicationaName, ReleaseNumber, a
     }
 }
 
-async function ApplicationandReleaseNumber(applicationaName, ReleaseNumber, authorization) {
+async function ApplicationandReleaseNumber(applicationaName, authorization) {
     let isApplicationNameExit = {};
```

Comment:
Correct the typo in the variable name from isApplicationNameExit to isApplicationNameExist. Modify it in all the used references.